.kokoro/build-with-appengine.sh

```diff
@@ -49,12 +49,10 @@ gcloud config set project $GCLOUD_PROJECT
 
 # Deploy the app
 gcloud app deploy --version $GAE_VERSION --no-promote --quiet
-if [ -e "worker.yaml" ]; then
-  gcloud app deploy worker.yaml --version ${GAE_VERSION} --no-promote --quiet
-fi
 
 
-# Test the deployed app
+# Install dependencies and run tests
+npm install
 npm test
 
 exit $?
```

Comment:
Do the node packages care about ending newlines?

Reply:
Probably not.